### PR TITLE
Release notes for version 3.0.0

### LINKED ISSUE #1231: Version 3.0.0 release notes
### Summary of Issue

Release notes need to be updated for version 3.0.0

### Issue Type

- [ ] Bug
- [ ] Enhancement
- [x] Chore

### Description

Release notes are needed for version 3.0.0, including all removed deprecations as backwards-incompatible changes.

### Example

N/A

### Environment

N/A

### PR DESCRIPTION
**Pull request by robfalck**
### Summary

Updated release notes for version 3.0.0.
Changed release notes to markdown so they're easier to read on github.

### Related Issues

- Resolves #1231 

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1,14 +1,115 @@
...
+we will support Python >= 3.6.  Functionally, 3.0.0 is identical to 2.10.1 with the exception
+that long-standing deprecations have been removed and will now result in errors.
+
+When upgrading to OpenMDAO 3.0.0, users should
+1. Get their models working with version 2.10.1
+2. Make necessary changes to remove any OpenMDAO-specific deprecation warnings
+3. Upgrade to OpenMDAO 3.0.0 and verify that their models are working as expected.
+
+Note: This is the last release of OpenMDAO 2.X. Updating to this release will be a
+critical step in preparing for the OpenMDAO 3.X releases
```
Comment (swryan) [inline release_notes.md]: I think this was meant to be deleted

File: release_notes.md [hunk 2]
```diff
@@ -1,14 +1,115 @@
...
+
+
+## Backwards Incompatible API Changes:
+
+The following are deprecated functionality since 2.0 that has been officially removed as of version 3.0.0:
+
+- <POEM 004> AkimaSplineComp and BsplineComp have been **removed**, replaced with SplineComp.
+
+- <POEM 008> The user is now **required** to set a value for the solver option `solve_subsystems`
+    on Newton and Broyden solvers. Previously the default was `False`, but you will now get
```
Comment (naylor-b) [inline release_notes.md]: will now --> would?
value is not --> value was not?

File: release_notes.md [hunk 3]
```diff
@@ -37,8 +138,7 @@ still may cause your results to differ when upgrading.
 - Change to the way SimpleGA does crossover to be more correct (was basically just mutuation before)
 
 
-New Features: 
--------------
+## New Features:
```
Comment (swryan) [inline release_notes.md]: I think we could delete the following line so that the 2.10 section is correct

File: release_notes.md [hunk 4]
```diff
@@ -1,14 +1,115 @@
...
+- Group attribute `nl_solver` is renamed `nonlinear_solver`.
+
+- Group attribute `ln_solver` is renamed `linear_solver`.
+
+- System attribute `metadata` is renamed `options`.
+
+- `units = 'unitless'` is removed.  Use `units = None` instead.
+
+- NewtonSolver attribute `line_search` is now `linesearch`.
+
```
Comment (Kenneth-T-Moore) [inline release_notes.md]: Add:

-  IndepVarComp: removed multi-variable specification via a list or tuple. Use add_output for each variable instead.
- Driver: removed method 'set_simul_deriv_color'. Use 'use_fixed_coloring' instead.
- Problem: 'vector_class' removed. Use 'distributed_vector_class' instead.
- pyOptSparseDriver: 'dynamic_simul_derivs' removed. Use 'declare_coloring' instead.

File: release_notes.md [hunk 5]
```diff
@@ -74,6 +71,13 @@ The following are deprecated functionality since 2.0 that has been officially re
 
 - NewtonSolver attribute `line_search` is now `linesearch`.
 
+- Multi-variable speicifcation via list or tuple to IndepVarComp is now invalid. Use add_output for each variable instead.
```
Comment (swryan) [inline release_notes.md]: typo: speicifcation

**Pull request merged**